src/doom/m_menu.c

```diff
@@ -603,6 +603,10 @@ enum
     crispness_demousetimer,
     crispness_sep_demos_,
 
+    crispness_sep_new_year,
```

Comment:
No chance this gets a switch in the menu. Sorry, but this feature has to remain secret. 😉

Reply:
It will New Year egg (: